src/helpers/providers/TranslationProvider.js

```diff
@@ -0,0 +1,27 @@
+import React, { createContext, useContext, useState } from "react";
+import { TranslatorProvider } from "react-translate";
+import translations from "../translations/translations";
+
+export const TranslationContext = createContext("");
+
+export const TranslationProvider = ({ children }) => {
+  const [translationModeCrypto, settranslationModeCrypto] = useState(true);
+
+  return (
+    <TranslatorProvider translations={translations}>
+      <TranslationContext.Provider
+        value={{
+          translationModeCrypto,
+          settranslationModeCrypto,
+          translationMode: translationModeCrypto ? "crypto" : "muggle"
```

Comment:
When we move this to Reason we can create a variant type for this, it will make the code nice and elegant, for sure :) 
https://reasonml.github.io/docs/en/variant